app/invocation/invocation_logs_model.tsx

```diff
@@ -18,6 +18,7 @@ export default class InvocationLogsModel {
   private logs = "";
   private responseSubscription: Subscription;
   private pollTailTimeout: number | null = null;
+	private stableLogLength = 0;
```

Comment:
nit: Add a comment like `// Length of the log prefix which has already been persisted. The remainder of the log is considered "live" and may be updated on subsequent fetches`